### DISCUSSION TITLE
Am i able to create coustom extensions for qute browser ?

### DISCUSSION THREAD

**Discussion by vishnudas-bluefox** [qutebrowser/qutebrowser#7425] [Ideas] [answered]
Am planning to create some custom extensions for my qutebrowser like an image reverse search engine, text-to-audio player, etc ...

was there any open documentation or guide for complete this task

**Comment by The-Compiler** [comment, marked as answer]
See [qutebrowser/userscripts.asciidoc at master · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/blob/master/doc/userscripts.asciidoc)
